css/style.css

```diff
@@ -170,17 +340,31 @@ p.content-description {
     font-size: 13px;
     font-weight: 400;
     line-height: 20px;
+} 
+.section-popular {
+    max-width: 1160px;
+    height: auto;
```

Comment:
what for do you use this auto everywhere?

Reply:
Removed.